services/src/main/java/org/exoplatform/addons/gamification/rest/ChallengeRest.java

```diff
@@ -222,12 +223,18 @@ public Response getAllChallengesByUser(
                                          @QueryParam("announcements")
                                          int announcementsPerChallenge,
                                          @ApiParam(
-                                             value = "Number of announcements per challenge",
-                                             required = false,
-                                             defaultValue = "0"
+                                             value = "term to search challenges with",
+                                             required = false
                                          )
                                          @QueryParam("term")
-                                         String term) {
+                                         String term,
+                                         @ApiParam(
+                                             value = "Challenge period filtering. Possible values: STARTED, NOT_STARTED, ENDED, ALL",
+                                             required = false
```

Comment:
```suggestion
                                             defaultValue = "ALL",
                                             required = false
```

Reply:
nope, not the same thing. One used for Swagger documentation and another for JAX-RS annotation & default value definition when null.